### ISSUE TITLE
Error importing private key to GUI BTCP wallet

### ISSUE THREAD

**Issue by Horeas** [BTCPrivate/bitcoin-private-full-node-wallet#31]
I get this error when i want to import private keys from a zcl electrum wallet to a GUI Wallet 1.0.4

![2018-04-25_3-56-49](https://user-images.githubusercontent.com/37069708/39231773-dd480a02-4873-11e8-8862-07bbaaea6095.jpg)

**Comment by dm8885555** [comment]
try this: https://github.com/ymgve/bitcoin_fork_claimer  Added support for BTCP segwit claiming 12 days ago

**Comment by Horeas** [comment]
Beats me how to use that . isnt there a guide?

**Comment by Ayms** [comment]
You can use https://github.com/Ayms/bitcoin-transactions also

**Comment by Horeas** [comment]
I put in there the private keys and they are then ready to be accepted in gui wallet?

**Comment by Horeas** [comment]
I had zcl before the fork and i could claim them in btcp electrum after i made a multisign wallet. Problem is that the old zcl wallet was a 2fa and now i dont have the posibility to cosign the btcp sending.

**Comment by Horeas** [comment]
Dont you have a video step by step. It seems very complicated!

**Comment by Ayms** [comment]
No, it's not so complicate, you can email me (see my github email)

**Comment by tommyranger** [comment]
Hello,
So I know what my Passphrase is for my BTC wallet, this I believe is the same as a Private Key. However when I attempt to import this same Passphrase into the BTCP wallet under the category 'Import One Private Key' I get a similar error message - please note the message is SIMILAR but not EXACT. See below:

![image](https://user-images.githubusercontent.com/36209208/39808609-4c7d86c8-5377-11e8-9617-18fba78e7316.png)

FYI I am using BTCP wallet version 1.0.3 and it is fully synced. I do not have any holding in ZCL, just BTC. Grateful for help regarding this error message and how I can rectify the issue.

Thank you,
Tom
